thorlcr/master/thgraphmanager.cpp

```diff
@@ -1001,12 +1002,52 @@ bool CJobManager::executeGraph(IConstWorkUnit &workunit, const char *graphName,
         } cBlock(activeTasks);
 
         {
+#ifdef _CONTAINERIZED
+            // collate pod distribution
+            VStringBuffer selector("thorworker-%s-%s", wuid.get(), graphName);
+            std::vector<std::vector<std::string>> pods = getPodNodes(selector.toLowerCase());
+            std::unordered_map<std::string, unsigned> podPerNodeCounts;
+            for (const auto &podNode: pods)
+            {
+                const std::string &node = podNode[1]; // pod is 1st item, node is 2nd
+                podPerNodeCounts[node]++; // NB: if doesn't exist is created with default value of 0 1st
```

Comment:
is getPodNodes likely to be called again?  If not it would be much more efficient for this code to be calculated directly instead of creating the intermediate vector